src/tests/system/tests/test_sssctl.py

```diff
@@ -191,32 +191,6 @@ def test_sssctl__reset_cached_timestamps_to_reflect_changes(client: Client, ldap
     assert "user1" not in res1.members
 
 
-@pytest.mark.importance("high")
-@pytest.mark.tools
-@pytest.mark.topology(KnownTopology.Client)
-def test_sssctl__check_invalid_option_name(client: Client):
```

Comment:
Commit message says
```
test_sssctl__check_invalid_option_name_in_snippet merges tests:
test_sssctl__check_invalid_option_name
...
```
But that's not the case - `test_sssctl__check_invalid_option_name` is simply removed.

Reply:
Probably claim is that `test_sssctl__check_invalid_option_name_in_snippet` ~= `test_sssctl__check_invalid_option_name`?